CMakeLists.txt

```diff
@@ -399,19 +399,31 @@ option( MIOPEN_DEBUG_FIND_DB_CACHING "Use system find-db caching" ON)
 # FOR HANDLING ENABLE/DISABLE OPTIONAL BACKWARD COMPATIBILITY for FILE/FOLDER REORG
 option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "Build with file/folder reorg with backward compatibility enabled" OFF)
 
-set( MIOPEN_INSTALL_DIR miopen)
-set( DATA_INSTALL_DIR ${CMAKE_INSTALL_DATAROOTDIR}/miopen )
+if(WIN32)
+    set( DATA_INSTALL_DIR bin )
+    set( DATABASE_INSTALL_DIR ${DATA_INSTALL_DIR})
+else()
+    set( MIOPEN_INSTALL_DIR miopen)
+    set( DATA_INSTALL_DIR ${CMAKE_INSTALL_DATADIR}/${MIOPEN_INSTALL_DIR} )
```

Comment:
Is it correct to change from `CMAKE_INSTALL_DATAROOTDIR` to `CMAKE_INSTALL_DATADIR` here?

Reply:
Yes, according to CMake documentation of GNUInstallDir
`DATADIR` inherits from `DATAROOTDIR` which is `share` on UN*X systems.
https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

Reply:
Both are read-only values, so we cannot change them in the CMakeLists.txt files.

Reply:
@apwojcik Okay, thanks.

Reply:
Sorry, my bad. The variables are supposed to hold the names of the directories for read-only files. They can be changed in CMakeLists.txt. The rest is proper. Both variables behave according to the GNU Coding Standard - https://www.gnu.org/prep/standards/html_node/Directory-Variables.html.